src/components/WorldListPage_WorldsForEnsOwnersFeature/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const fromBytesToMegabytes = (bytes: number) => {
+  return bytes / 1000000
```

Comment:
https://github.com/decentraland/builder/compare/feat/blocked-state#diff-0f09525ed78eee410b675c0f050c346cf3bb2245e384a1c1fc78637703870afeR2 

Updated to this in a future PR

```
return bytes / 1024 / 1024
```